py/janitor/site/simple.py

```diff
@@ -705,12 +705,18 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await private_runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(private_runner, args.host, port=args.port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.port)
+    logging.info("Listening privately on %s:%s", args.host, args.port)
```

Comment:
This isn't privately listening (it's the same listen address) - it's the private endpoints that are available on that IP.

Reply:
Sorry, I'm not following.

From my understanding: `address == host + port`
Yes, the host is always the same, but the ports are different (one public, one private).
As a result, its not the same listen address?

Reply:
Otherwised, based on the orginal output, how do you know which port is for which?

Reply:
Right, but "privately listening" implies that this is restricted to e.g. private space somehow (e.g. 127.0.0.1) - which it is not.

Reply:
Is there a term/phrase that you think would be a better fit?

Reply:
```suggestion
    logging.info("Admin API listening on %s:%s", args.host, args.port)
```

Reply:
Also agree - added to the other janitor-* services too.

Reply:
Big thanks!